src/status_im/contexts/preview/quo/community/channel_actions.cljs

```diff
@@ -34,6 +34,17 @@
        :customization-color :blue
        :icon                :i/muted}]}]
    [rn/view {:height 50}]
+   [quo/channel-actions
+    {:actions
```

Comment:
Is it possible to add accessibility labels for QA team?

Reply:
done